qmlui/qml/virtualconsole/VCSliderItem.qml

```diff
@@ -230,7 +230,7 @@ VCWidgetItem
                 if (Qt.platform.os === "android")
                     presetImageBox.source = cngResource
                 else
-                    presetImageBox.source = "file:/" + cngResource
```

Comment:
I noticed you removed the extra slash in a few places. Which operating system are you on? I suspect this won't work on every supported platform.

Reply:
It is Linux AppImage. Gobos don't show otherwise. The path is otherwise searched somewhere in the root directory rather than in the relative filesystem.